python/lsst/analysis/ap/utilities.py

```diff
@@ -0,0 +1,70 @@
+# This file is part of analysis_ap.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__all__ = ["makeSimbadLink"]
+
+from IPython.display import display, Markdown
+import astropy.coordinates as coord
+from astroquery.simbad import Simbad
+import astropy.units as u
+
+
+def makeSimbadLink(source, arcsec_radius=3.0, make_table=False):
+    """Search Simbad for associated sources within a 3 arcsecond region
+        Maximum search radius should not be greater than 5 arcseconds.
+
+    Parameters
+    ----------
+    source : 'pandas.core.series.Series'
```

Comment:
I suggest that instead of a pandas series you simply provide the ra and dec as positional arguments--it will make the function more flexible in other contexts.

Reply:
Changed to accept ra and dec.